src/content/release-notes.mdx

```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
```

Comment:
Let's add a link to the nginx release notes:
https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2